jenkins/jobs/jobs.yaml

```diff
@@ -175,6 +175,49 @@
       - osci-swift-upload-logs-with-console-plugin:
           upload_source: 'test.log logs'
 
+- job-template:
+    name: 'dsvm-tempest-vgpu'
```

Comment:
this is not tempest test.

Reply:
Use this name to reuse skip option. or we need to made another copy for vgpu.
  - name: ^dsvm-tempest.*$
    skip-if:
      - project: ^openstack/nova$
        all-files-match-any:
          - ^.*\.rst$
          - ^doc/.*$
          - ^nova/tests/.*$
          - ^nova/virt/baremetal/.*$
          - ^nova/virt/hyperv/.*$
          - ^nova/virt/ironic/.*$
          - ^nova/virt/vmwareapi/.*$
          - ^nova/virt/libvirt/.*$
          - ^tools/.*$
          - ^tox.ini$
      - project: ^openstack/neutron$
        all-files-match-any:
          - ^.*\.rst$
          - ^doc/.*$
          - ^neutron/locale/.*$
          - ^neutron/tests/.*$
          - ^releasenotes/.*$
          - ^tools/.*$
          - ^tox.ini$
    parameter-function: single_use_node